woodwork/logical_types.py

```diff
@@ -71,7 +72,16 @@ def transform(self, series):
         return series
 
     def validate(self, series, return_invalid_values=True):
-        return
+        """Validates that a logical type is consistent with the series dtype. Performs additional type
+        specific validation, as required."""
+        valid_dtype = self._get_valid_dtype(type(series))
```

Comment:
We should be testing this method to confirm that logical types without any further validation method defined are properly checking that their dtypes are the correct type. Something like 

```
s = pd.Series([1,2,3])
s.ww.init(logical_type=Integer)

changed_dtype_series = s.astype('string')

with pytest.raises(TypeValidationError, match=match):
     Integer.validate(changed_dtype_series)
```